index.html

```diff
@@ -26,6 +26,20 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+
+      <div id="mobile-nav" class="invisible">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <div>
+            <li><a class="mobile-link" href="#projects">Portoflio</a></li>
+            <li><a class="mobile-link" href="#about-m">About</a></li>
+            <li>
+              <a class="mobile" href="#contact">Contact</a>
+            </li>
+          </div>
```

Comment:
- [ ] Please, make sure to always wrap your `<li>` inside of an `<ul>` or `<ol>` instead of a `<div>`. That'll improve the HTML structure of your project & makes it more accessible.